.circleci/config.yml

```diff
@@ -33,7 +33,7 @@ jobs:
   integration-tests:
     executor: python/default
     working_directory: ~/repo
-    parallelism: 10
+    parallelism: 2
```

Comment:
This is a diff from 10 -> 2, simply because I tried to add parallelism in the staging branch. I incorrectly assumed this would be a couple line change to add parallelization :)